tools/nvqpp/nvq++.in

```diff
@@ -260,6 +268,18 @@ gpu_found=$(query_gpu)
 if ${gpu_found} && [ -f "${install_dir}/lib/libnvqir-custatevec-fp32.so" ]; then
 	NVQIR_SIMULATION_BACKEND="custatevec-fp32"
 fi
+# Check environment variable - overrides the default
+# NOTE: Invalid values are ignored; ASKME - show warning?
```

Comment:
showing a warning in that case sounds good

Reply:
On second thoughts, it may be inconsistent behavior showing warning in only this specific case, there aren't other similar instances, as far as I can tell. I will record it documentation.